privacyidea/lib/tokens/emailtoken.py

```diff
@@ -326,6 +328,16 @@ def _get_email_text_or_subject(options,
                 message = messages[0]
 
         message = message.format(challenge=options.get("challenge"))
+        if message.startswith("file:"):
+            # We read the template from the file.
+            try:
+                f = open(message[5:], "r")
+                message = f.read()
+                f.close()
+            except Exception as e:  # pragma: no cover
+                log.warning(u"Failed to read email template: {0!r}".format(e))
```

Comment:
In case the file could not be opened, the email message will be the action value of the policy, e.g. ``file:/etc/privacyidea/email-template.txt`` -- we could instead revert to a default value?

Reply:
right.